lighthouse-core/gather/driver.js

```diff
@@ -648,7 +648,6 @@ class Driver {
     const waitForNavigated = options.waitForNavigated || false;
     const waitForLoad = options.waitForLoad || false;
     const passContext = /** @type {Partial<LH.Gatherer.PassContext>} */ (options.passContext || {});
-    const disableJS = passContext.disableJavaScript || false;
```

Comment:
🎉 🎉 

should also delete in [`PassContext`](https://github.com/GoogleChrome/lighthouse/blob/f71cfc0ab431afbc2a2a4225c0222ba0fcda33e5/types/gatherer.d.ts#L39)

🎉 🎉

Reply:
will not miss you `disableJavaScript` :)